examples/pom.xml

```diff
@@ -0,0 +1,106 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.apache.sedona</groupId>
+  <artifactId>sedona-examples</artifactId>
+  <version>1.6.0</version>
+  <name>Sedona : Examples</name>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>flink-sql</module>
+    <module>spark-sql</module>
+  </modules>
+
+  <properties>
+    <log4j.version>2.17.2</log4j.version>
+    <slf4j.version>1.7.36</slf4j.version>
+    <geotools.version>28.2</geotools.version>
+    <flink.version>1.19.0</flink.version>
+    <scala.compat.version>2.12</scala.compat.version>
+  </properties>
+
+  <!-- add OSGeo Repositories for examples-->
+  <repositories>
```

Comment:
Please don't move these dependencies to the parent pom. I think we don't even need to have a parent project for the examples. The reasons are 

1. Spark and Flink have very different dependecies. It is hard to find a common set of log4j that work for all cases.
2. Each Spark/Flink example project should be as standalone as possible. This way, a new Sedona user can just pick an example project and use it as a template to develop his/her application.

A complicated project structure makes new user hard to get started.

Reply:
sounds good. removed the parent project.